docs/web-apps/automated-testing/cypress/yaml/v1.md

```diff
@@ -334,14 +334,57 @@ npm:
 
 ---
 
-### `registry`
```

Comment:
I don't think we should delete it just like that. Because `registries` would only be supported in certain framework + saucectl versions. There's no explanation as to what these are in saucectl or here.

Reply:
I restored the `registry` field + add a note that this is supported up to <current version>. Then should use `registries`.